sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -499,6 +501,14 @@ object OverwritePartitionsDynamic {
   }
 }
 
+case class DescribeTable(
+    catalog: TableCatalog,
+    ident: Identifier,
+    isExtended: Boolean) extends Command {
+  override lazy val output = DescribeTableSchemas.DESCRIBE_TABLE_ATTRIBUTES
+  override lazy val schema = DescribeTableSchemas.DESCRIBE_TABLE_SCHEMA
```

Comment:
by default schema is `StructType.fromAttributes(output)`, so we don't need to override it.